src/domain/feature.js

```diff
@@ -3,22 +3,23 @@
  */
 
 import { isString } from '../utils/checks';
-import { Geometry } from './geometry';
+import { BaGeometry } from './geometry';
 
 /**
  * A feature.
```

Comment:
Suggestion: 'A feature in BA context' - to clarify that there are also other 'feature' (OpenLayers).